sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -35,6 +36,7 @@
  * @see Configuration
  */
 public class ClientLogger {
+    private static final Pattern CRLF_PATTERN = Pattern.compile("[\r\n]");
```

Comment:
Are we looking to match `\r` or `\n` or `\r\n`? The regex is currently matching the former.

Reply:
it is either of \r or \n.
The solution is recommended here: https://find-sec-bugs.github.io/bugs.htm#CRLF_INJECTION_LOGS